query-node/schemas/membership.graphql

```diff
@@ -47,6 +47,9 @@ type MemberMetadata @entity {
 
   "Social media handles, email address..."
   externalResources: [MembershipExternalResource] @derivedFrom(field: "memberMetadata")
+
+  isVerified: Boolean!
```

Comment:
Since this is not in a separate entity anymore. Please rename it to something more explicit like `isVerifiedValidator`